### ISSUE TITLE
false positive on Intel CPU model 06-6a-06

### ISSUE THREAD

**Issue by gaowayne** [speed47/spectre-meltdown-checker#430]
hello checker team, 
after confirmed intel expert, checker make a mistake here. 
on CPU model 06-6a-06, our latest FW version is just 0d000331 from the https://github.com/intel/Intel-Linux-Processor-Microcode-Data-Files

but checker complain need  0d000332, we do not have 332 release yet. could you please help fix the checker tool ASAP?

```
0000000 00000001 0d000331 12032021 000606a6
0000020 85c35e65 00000001 00000087 000473d0
0000040 00047400 00000000 00000000 00000000
0000060 00000000 000000e0 00030001 0d000331
0000100 00000007 00011c00 20211202 00008880
```

**Comment by speed47** [comment]
0d000332 exists in MCEdb, probably because it has been seen somehere in the wild, as an alpha or test version or something.
Now, when we have input from the Intel repo about a firmware, we'll take it, even if MCEdb also has info about the same CPU model, and even if it's more recent (to avoid reporting existence of firmwares that are not readily available).
